.github/ISSUE_TEMPLATE/01_report_a_issue.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: 🐞 Report an issue
+description: Problems and issues in EPAM AI DIAL chat
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      # yamllint disable rule:line-length
+      value: "
+        <img src='https://www.gravatar.com/avatar/2937817a82464ae07b138d8b57a7ccd4?s=40' align='left' width='40' height='40' style='margin-right:10px'>
+        Thank you for finding the time to report the problem!
+
+        We really appreciate the community's efforts to improve EPAM AI DIAL chat.
+        <br clear='left'/>"
+      # yamllint enable rule:line-length
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: EPAM AI DIAL version
+      # yamllint disable rule:line-length
+      description: >
+        On what version of EPAM AI DIAL are you currently experiencing the issue? You can find the version in the footer of page (e.g. "Version 0.4.2").
```

Comment:
It's better to rename to 'On what version of AI DIAL CHAT ...', because version of entire dial is different for now